gen/__init__.py

```diff
@@ -569,7 +569,12 @@ def generate(
     # of all arguments would want to include itself).
     # Explicitly / manaully setup so that it'll fit where we want it.
     # TODO(cmaloney): Make this late-bound by gen.internals
-    argument_dict['expanded_config'] = textwrap.indent(json_prettyprint(argument_dict), prefix='  ' * 3)
+    argument_dict['expanded_config'] = textwrap.indent(
```

Comment:
Seems acceptable for now. Practically the late config being yaml rather than json is also due to escaping issues. It's tech debt which we can reasonably push off for now.